src/Utilities/TypeTraits/CreateGetStaticMemberVariableOrDefault.hpp

```diff
@@ -0,0 +1,35 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <type_traits>
+
+/*!
+ * \ingroup TypeTraitsGroup
+ * \brief Generate a metafunction that will retrieve the specified
+ * `static constexpr` member variable, or, if not present, a default.
```

Comment:
Would it be reasonably straight-forward to make this macro work for type aliases as well?